core/src/scene/sceneLoader.h

```diff
@@ -27,6 +28,7 @@ struct MaterialTexture;
 struct Filter;
 
 using Mixes = std::vector<YAML::Node>;
+using UniqueStyles = std::unordered_set<std::string>;
```

Comment:
I mean that I don't think these type aliases are needed. I think the code would be clearer if the actual type were used; the intent of the relevant variables is already expressed in their names.